attributes/default.rb

```diff
@@ -4,7 +4,7 @@
 #
 # Copyright 2014-2016, Bloomberg Finance L.P.
 #
-
+default['go']['version'] = '1.7'
```

Comment:
Another option would be to update the kitchen tests to specify go 1.7, and then have a check to make sure go 1.7 is installed?  It's listed as a flat our requirement on the consul website.  When I did a clean checkout, tests were completely failing to do any git based installs.  This was one of the things I'd seen to fix it (the other was that zip was missing).